vng_api_common/serializers.py

```diff
@@ -264,3 +264,34 @@ def update(self, instance, validated_data):
                 continue
             setattr(instance, name, validated_data.pop(name))
         return super().update(instance, validated_data)
+
+
+class LengthHyperlinkedRelatedField(serializers.HyperlinkedRelatedField):
+    def __init__(self, *args, **kwargs):
+        self.min_length = kwargs.pop("min_length", None)
+        self.max_length = kwargs.pop("max_length", None)
+        kwargs.setdefault("allow_null", True)
```

Comment:
this should _only_ be done if the field is not required

```suggestion
        kwargs.setdefault("allow_null", not kwargs.get("required", True))  # please check what the default value is in DRF
```